packages/admin-e2e-tests/src/specs/activate-and-setup/complete-onboarding-wizard.ts

```diff
@@ -517,9 +517,70 @@ const testSubscriptionsInclusion = () => {
 	} );
 };
 
+const testBusinessDetailsForm = () => {
+	describe( 'A store that is selling elsewhere will see the "Number of employees” dropdown menu', () => {
+		const profileWizard = new OnboardingWizard( page );
+		const login = new Login( page );
+
+		beforeAll( async () => {
+			await login.login();
+		} );
+
+		afterAll( async () => {
+			await deactivateAndDeleteExtension( 'woocommerce-payments' );
```

Comment:
Nice clean-up!  Are there other extensions we need to worry about deactivating or is this the only one of concern?

Reply:
I couldn't find any other extension we should deactivate, just WC-Pay.